cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -19,6 +20,9 @@
 {% if pillar['enable_deployments'] is defined -%}
   {% set enable_deployments = "--enable-deployment-controller=" + pillar['enable_deployments'] -%}
 {% endif -%}
+{% if grains.cloud in [ 'gce' ] and grains.cloud_config is defined -%}
```

Comment:
apologies if this is obvious to people who grok salt -- I don't see why cloud_config needs to be defined in addition to the cloud==gce test?

But then, I'm not sure why this is behavior we'd only want for GCE -- presumably AWS and GKE and bare metal clusters should also not fall over as the number of terminated pods goes to infinity?  What's the justification for GCE-only?